books/free-programming-books-id.md

```diff
@@ -126,6 +126,7 @@
 
 ### Machine Learning
 
+* [Machine Learning] (https://www.tutorialspoint.com/machine_learning/machine_learning_tutorial.pdf) - Tutorials Point (PDF)
 * [Tutorial Belajar Machine Learning Dasar \| Python Scikit-Learn](https://www.youtube.com/playlist?list=PL2O3HdJI4voHNEv59SdXKRQVRZAFmwN9E) - Indonesia Belajar
```

Comment:
There are a couple of youtube links in this books file. They should be moved to corresponding course files later

Reply:
#9177